docs/modules/ROOT/pages/utils/_class_hashes.adoc

```diff
@@ -1,5 +1,5 @@
 // Version
-:class-hash-cairo-version: https://crates.io/crates/cairo-lang-compiler/2.3.1[cairo 2.3.1]
+:class-hash-cairo-version: https://crates.io/crates/cairo-lang-compiler/2.4.4[cairo 2.4.4]
```

Comment:
```suggestion
:class-hash-cairo-version: https://crates.io/crates/cairo-lang-compiler/2.4.1[cairo 2.4.1]
```

Good catch! Though, the listed hashes are from Cairo v2.4.1 with the release (v2.4.4 outputs different hashes)

I think we should add compiling the class hashes as a TODO item in RELEASING.md because this is really easy to overlook. WDYT?

Reply:
the released version is fine, it says 2.4.1 which matches the class hash. The problem is in the `main` branch, we overlooked a cairo lang bump without class hash update. The item should be added to the PR checklist maybe? Or automating it for good.

Reply:
i just updated the class hashes too to fix the issue

Reply:
Automating would definitely be preferable. I'm not opposed to adding it to the PR checklist in the mean time